convert/src/vertex_to_bd/vertex_to_badgerdoc_use_case.py

```diff
@@ -56,29 +75,40 @@ def download_labelstudio_from_s3(
                 s3_input_annotation.path,
                 str(input_file),
             )
-            return LabelStudioModel.parse_file(input_file)
+            result = LabelStudioModel.parse_file(input_file)
+            return result
 
-    def upload_badgerdoc_to_s3(
-        self,
-        s3_output_tokens,
-        s3_output_pdf,
-        s3_output_annotations,
+    def get_output_tokens_path(self):
+        return f"files/{self.s3_output_file_id}/ocr/tokens.json"
+
+    def get_output_pdf_path(self):
+        return f"files/{self.s3_output_file_id}/{self.s3_output_file_id}.pdf"
+
+    def get_output_annotations_path(self, importjob_id_created):
+        return f"annotation/{importjob_id_created}/{self.s3_output_file_id}/annotation.json"
+
+    def upload_badgerdoc_format_to_s3(
+        self, importjob_id_created
     ) -> None:
         with tempfile.TemporaryDirectory() as tmp_dirname:
             tmp_dirname = Path(tmp_dirname)
```

Comment:
To make code more maintainable I would not assing different types to the same var

Reply:
I've put typehints there, if that's what you mean.